pipelines/coref/README.md

```diff
@@ -0,0 +1,54 @@
+<!-- SPACY PROJECT: AUTO-GENERATED DOCS START (do not remove) -->
+
+# 🪐 spaCy Project: Training a spaCy Coref Model
+
+This project trains a coreference model for spaCy using OntoNotes.
+
+
+## 📋 project.yml
+
+The [`project.yml`](project.yml) defines the data assets required by the
+project, as well as the available commands and workflows. For details, see the
+[spaCy projects documentation](https://spacy.io/usage/projects).
+
+### ⏯ Commands
+
+The following commands are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run).
+Commands are only re-run if their inputs have changed.
+
+| Command | Description |
+| --- | --- |
+| `prep-data` | Rehydrate the data using OntoNotes |
+| `preprocess` | Convert the data to spaCy's format |
+| `train-cluster` | Train the clustering component |
+| `prep-span-data` | Prepare data for the span predictor component. |
+| `train-span-predictor` | Train the span predictor component. |
+| `assemble` | Assemble parts into complete coref pipeline. |
+| `eval` | Evaluate model on the test set. |
+
+### ⏭ Workflows
+
+The following workflows are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run)
+and will run the specified commands in order. Commands are only re-run if their
+inputs have changed.
+
+| Workflow | Steps |
+| --- | --- |
+| `prep` | `prep-data` &rarr; `preprocess` |
+| `train` | `train-cluster` &rarr; `prep-span-data` &rarr; `train-span-predictor` &rarr; `assemble` |
+| `all` | `prep-data` &rarr; `preprocess` &rarr; `train-cluster` &rarr; `prep-span-data` &rarr; `train-span-predictor` &rarr; `assemble` |
+
+### 🗂 Assets
+
+The following assets are defined by the project. They can
+be fetched by running [`spacy project assets`](https://spacy.io/api/cli#project-assets)
+in the project directory.
+
+| File | Source | Description |
+| --- | --- | --- |
+| `assets/` | Git | CoNLL-2012 scripts and dehydrated data. |
+| `/home/USER/ontonotes5/data` | Local | Ensure you have a local copy of OntoNotes: https://catalog.ldc.upenn.edu/LDC2013T19 |
```

Comment:
I was copying this to a clean system and wondering: it doesn't work if you only store the English data, right? I was hoping it would so I didn't need to copy all of it, but `prep-data` errors with 
```
$ spacy project run prep-data
"Rehydrating data... This will take roughly 20 minutes"
----------------------------------------------------------------------------------------------------

could not find the gold parse [c://data/ontonotes5/data/arabic/annotations/nw/ann/00/ann_0001.parse] in the ontonotes distribution ... exiting ...

----------------------------------------------------------------------------------------------------
cat: 'assets/conll-2012/v4/data/development/data/english/annotations/*/*/*/*.v4_gold_conll': No such file or directory
```

Just wondering if maybe there's an easy fix to allow users to have only the English data, as I assume this project only  uses the English data, too.

Reply:
(this is not a showstopper though!)

Reply:
I had not checked that, thanks for noting it! We definitely only need the English data, so I'll see how hard it would be to modify the old scripts to deal with this.

Reply:
I have not tested running the modified script yet, but I added an optional `sed` command to the `project.yml` that I'm pretty sure can be used to only do the English files.

Reply:
I verified this works and turned it on by default since it speeds up preprocessing.